targets/TARGET_NORDIC/TARGET_NRF5/qspi_api.c

```diff
@@ -185,30 +233,52 @@ qspi_status_t qspi_read(qspi_t *obj, const qspi_command_t *command, void *data,
     }
 }
 
-// they provide 2 functions write or nrf_drv_qspi_cinstr_xfer
-// nrf_drv_qspi_cinstr_xfer seems like it accepts simplified config that is very simplified
-// and might not be useful for us. 
-// write on other hand, needs to write some data (errors if buffer is NULL!)
-qspi_status_t qspi_write_command(qspi_t *obj, const qspi_command_t *command)
+qspi_status_t qspi_command_transfer(qspi_t *obj, const qspi_command_t *command, const void *tx_data, size_t tx_size, void *rx_data, size_t rx_size)
 {
-    // use simplified API, as we are sending only instruction here
-    nrf_qspi_cinstr_conf_t config;
-    config.length = NRF_QSPI_CINSTR_LEN_1B; // no data
-    config.opcode    = command->instruction.value;
-    config.io2_level = false;
-    config.io3_level = false;
-    config.wipwait   = false;
-    config.wren      = false;
-
-    // no data phase, send only config
-    ret_code_t ret = nrf_drv_qspi_cinstr_xfer(&config, NULL, NULL);
-    if (ret == NRF_SUCCESS ) {
-        return QSPI_STATUS_OK;
-    } else {
+    ret_code_t ret_code;
+    uint32_t i;
+    uint8_t data[8];
+    uint32_t data_size = tx_size + rx_size;
+ 
+    nrf_qspi_cinstr_conf_t qspi_cinstr_config;
+    qspi_cinstr_config.opcode    = command->instruction.value;
+    qspi_cinstr_config.io2_level = false;
+    qspi_cinstr_config.io3_level = false;
+    qspi_cinstr_config.wipwait   = false;
+    qspi_cinstr_config.wren      = false;
+ 
+    if (data_size < 9)
+    {
```

Comment:
`{` on the same line in this function for every condition/loop

Reply:
Will fix.